src/thunderbots/software/radio_communication/main.cpp

```diff
@@ -1,61 +1,37 @@
 #include <ros/ros.h>
 #include <thunderbots_msgs/Primitive.h>
 #include <thunderbots_msgs/PrimitiveArray.h>
+#include <thunderbots_msgs/World.h>
 
 #include "ai/primitive/primitive.h"
 #include "ai/primitive/primitive_factory.h"
-#include "geom/point.h"
-#include "grsim_communication/grsim_backend.h"
 #include "util/constants.h"
 #include "util/logger/init.h"
 
-// Member variables we need to maintain state
-// They are kept in an anonymous namespace so they are not accessible outside this
-// file and are not created as global static variables.
-namespace
-{
-    // A vector of primitives. It is cleared each tick, populated by the callbacks
-    // that receive primitive commands, and is processed by the backend to send
-    // the Primitives to the robots using the radio.
-    std::vector<std::unique_ptr<Primitive>> primitives;
-}  // namespace
-// Callbacks
 void primitiveUpdateCallback(const thunderbots_msgs::PrimitiveArray::ConstPtr& msg)
 {
+    std::vector<std::unique_ptr<Primitive>> primitives;
     thunderbots_msgs::PrimitiveArray prim_array_msg = *msg;
     for (const thunderbots_msgs::Primitive& prim_msg : prim_array_msg.primitives)
     {
         primitives.emplace_back(AI::Primitive::createPrimitiveFromROSMessage(prim_msg));
     }
```

Comment:
Should we be doing something else with these primitives?

Reply:
Yes, but that's up to @henrymliu 's later PRs

Reply:
Ok, can we leave a TODO with an issue reference here?

Reply:
Henry's PRs are merged and I've rebased on top of them so this is no longer an issue.